### ISSUE TITLE
Fail to run Transmission after update to 2.84-11

### ISSUE THREAD

**Issue by nilsgruger** [SynoCommunity/spksrc#1972]
Hi, I recently updated Transmission from 2.84-10 to 2.84-11 on my 1515+ with DSM 5.2-5644 Update 2. However, after the update, it won't start. It just says "Failed to run the package service." when trying to run it from the package center. I suspect this is not a known bug as the new version was released, but I hope that someone might have an idea how this could happen. Does anyone have an idea why it won't run?

**Comment by Dr-Bean** [comment]
Sometime in the past, you have changed the user or permissions associated with the `transmission` user to something else. This breaks our package upgrades. Either revert those changes, or do a clean install of the package.

**Comment by Dr-Bean** [comment]
Closing. As for your permissions problem, don't change the package user, there's no need for that. Use the `sc-download` group, or the `users` group to set permissions on the dirs you want to use (and check that the parent dirs have sufficient permissions too).

**Comment by nilsgruger** [comment]
Do you mean that I should just set the transmission user to belong to the sc-download group? I don't understand how to solve the permissions problem. When installing transmission I pointed out /volume1/Downloads as the directory for downloads. This obviously existed before as I just reinstalled transmission. I changed such that sc-download and the transmission user have access to that directory, but still I have the permission problem.

**Comment by Dr-Bean** [comment]
Apparently, you have created a separate `transmission` user yourself? Don't do that. As I said, no need for that, and besides, it's likely to break the package installer.

All of our packages create their own user with the correct settings during installation...in this case, conveniently called `transmission`. Note that those users are invisible via the GUI. That is on purpose.

The Transmission installer adds that user to the `users` group, as well as the `sc-download` group. It also sets the appropriate permissions for the directory you chose during the wizard (`volume1/downloads` by default, note that linux is case-sensitive).

In other words: if you'd start with a clean setup, no users or other changes, and keeping the defaults, you'd do two things:
- Create a shared folder called `downloads` on volume1;
- Followed by simply installing the package. You're good to go.

If you want to change that directory, or use a subdirectory of a shared folder (as in `/volume1/downloads/subdir` for example), then some manual permissions setting is needed. That's where the `sc-download` group, or the `users` group come in, both can be adjusted via the DSM GUI.

**Comment by nilsgruger** [comment]
I didn't know that. Is there any official guide of how to do a proper install? I just followed some guide which at that time solved my problems.

When I reinstalled transmission, I was asked to set a user name and password for, I think, UI. I think It also said that default was admin. So what user is this? If I want to access transmission from transmission remote GUI, what username/password should I use? I tried using the same as I entered when installing but that gives me "access denied".

**Comment by Dr-Bean** [comment]
No guide, because it's as simple as pushing the `Install` button. You're guided through a few questions via a wizard where you can adjust things to suit your needs and after installation is complete, you can start using the software immediately.
Other than that, I don't know what you'd want to read in a guide?

As for the user/pass: The wizard will tell you to enter a user/pass for the webinterface. Default is indeed admin/admin, but you can change it to whatever you want. It's possible that some special characters don't work, but I've never seen any issues reported on that.

Sidenote: Improved permissions handling of subdirs will be added in the near future via https://github.com/SynoCommunity/spksrc/issues/1932).

**Comment by nilsgruger** [comment]
I think I was confused by the fact that the user/pass I created in the wizard did not work. Therefore I assumed I needed to do something else as I recall from the guides I read some time back. Now I tried again with the default, and then I added group access to the shared volume and it seems to work. Thanks for the help! It seems like those "guides" were a waste of time... This issue can be closed again.

**Comment by GSX550ES** [comment]
Brand new fresh install from a new unit bought today with new hard drives, added the community, pressed Install on Transmission, under Basic configuration in the 3 directories asked for I put /volume1/downloads
/volume/incoming
/volume1/parts, 

I then clicked Next, on next page I put Username as Transuser Password (password) clicked next, clicked apply, & go the error message Failed to install "Transmission"

Nothing else had been done to the unit
